packages/tooltip/src/components/popover/Popover.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  Box,
+  Clickable,
+  ThemeColorField,
+  useBoolean,
+  useOnClickOutside,
+  useThemeFields
+} from "@stenajs-webui/core";
+import * as PopperJS from "popper.js";
+import * as React from "react";
+import { ReactNode, useMemo, useRef } from "react";
+import { Manager, Popper, Reference } from "react-popper";
+import { Arrow } from "./Arrow";
+
+export type PopoverTriggerType = "hover" | "click";
+
+type PopoverContentFunc = (args: PopoverContentFuncArgs) => ReactNode;
+
+interface PopoverContentFuncArgs {
+  show: () => void;
+  hide: () => void;
+}
+
+export interface PopoverProps {
+  placement?: PopperJS.Placement;
+  trigger?: PopoverTriggerType;
+  content?: ReactNode | PopoverContentFunc;
+  children?: ReactNode | PopoverContentFunc;
+  background?: ThemeColorField | string;
+  zIndex?: number;
+}
+
+export function Popover({
+  placement,
+  trigger = "hover",
+  content,
+  children,
+  background = "white",
+  zIndex = 2147483647
+}: PopoverProps) {
+  const [showing, show, hide] = useBoolean(false);
+  const outerRef = useRef(null);
+  useOnClickOutside(outerRef, () => {
+    if (trigger === "click") {
```

Comment:
I still think we should provide a prop to make this behaviour optional

Reply:
Done!